src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -190,11 +197,22 @@ AsyncReplResult<> RaftReplDev::replace_member(replica_id_t member_out_uuid, repl
                     } else {
                         RD_LOGI("Replace member removed member={} from group_id={}", member_out, group_id_str());
                     }
+
+                    // Revert the quorum size back to 0.
+                    reset_quorum_size(0);
```

Comment:
we better revert the quorum size after step 2?

Reply:
We cant do any leader related operation. add/remove/propose.

Reply:
After step 2 we have 2 members. Why we still need this quorumn size to move forward?

Reply:
New follower not added to quorum till the baseline is done right ?

Reply:
Sorry I got it we are in 2(live)/4(all),

Reply:
But you don't proceed to step3 before new member added into quorumn right?

Reply:
No its async operation. add_srv in nuraft only the leader accepted the request. CM will use hearbeats  with LSN(snapshot sync shard lsn) to know if its added or not.